### ISSUE TITLE
Conflicting release version in releases tab, and missing version tarball

### ISSUE THREAD

**Issue by solbu** [philpem/printer-driver-ptouch#33]
Hi.
I am the Mageia packager for this driver.
In my attempt at packaging the new version, I find that in the Release tab, the head title says «v1.7».
But both the «v1.7» and the sources links, link to «v1.6.1», so I suspect the «v1.7» is a typo.

So, which version is correct? :-)

Also, this time around you didn't prepare a versioned tarball like the other releases, which packagers like myself use for packaging.

**Comment by philpem** [comment]
Oops!... on several counts!  Will address this and #34 at the same time.

I just hit "tag and release" on Github and didn't realise packagers were using the prepped tarballs. You may feel free to read this as "I was being lazy".

The version number was indeed a mistake on my part, and it was wrong in other places too.

Thanks for raising this, I'll do a re-release shortly and it should be correct this time!

**Comment by solbu** [comment]
I thought the version number it was a typo. :-)

And yes, packagers prefer release tarball's over anything else.
We _do_ use tags when tarball's is not available, but tarballs are the prefered option as the archive name is important for the distros package automation tools.

**Comment by philpem** [comment]
I figured you'd be using the Github autogenerated tarballs but your reasoning sounds pretty good.

Steps taken to fix this... 

* I've written a [Wiki page](https://github.com/philpem/printer-driver-ptouch/wiki/HOWTO:-Make-a-release) to document the release tarball process, which should hopefully reduce the "activation energy" for me to do releases.
* [Version 1.7 has been released](https://github.com/philpem/printer-driver-ptouch/releases/tag/v1.7), adding support for three new printers and fixing a bug with the QL-570.

Congratulations on being the first distribution to package the new version  :+1:  :tada: 
I hadn't heard of Mageia - interesting to hear Mandriva is still going in spirit. It was one of the first Linux distros I used, way back in the day. Lots of very happy memories :)
